tpmutil/run.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2018, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package tpmutil provides common utility functions for both TPM 1.2 and TPM 2.0 devices.
+package tpmutil
+
+import (
+	"encoding/binary"
+	"errors"
+	"io"
+)
+
+const maxTPMResponse = 4096
+
+// RunCommand executes cmd with given tag and arguments. Returns TPM response
+// body (without response header) and response code from the header. Returned
+// error may be nil if response code is not RCSuccess, caller should check
```

Comment:
tiny language nit: ", caller should check both" is a comma splice. I'd prefer a semi-colon instead of a comma. But I recognize that this is already a comma splice of sentence fragments and is in a highly stylized form, so I'm OK if you won't want to change this.

Reply:
Done